components/Dialogs/src/MDCAlertController.m

```diff
@@ -18,9 +18,9 @@
 
 #import <MDFInternationalization/MDFInternationalization.h>
 
-#import "MDCDialogTransitionController.h"
 #import "MaterialButtons.h"
 #import "MaterialTypography.h"
+#import "MDCDialogTransition.h"
```

Comment:
Super-nit: Please order "MDC" before "Mat" for consistency with most of the MDC-iOS code.  I believe we tend to go with ASCII ordering for imports.

Reply:
Done.